cmd/plugin/plugin.go

```diff
@@ -32,31 +33,25 @@ func InstallCommand(out output.Output) *cobra.Command {
 		Use:   "install",
 		Short: "Installs a tool for a specific plugin",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			out.V(6).Info(fmt.Sprintf("args: %v", args))
-			return nil
-		},
-	}
-}
+			avm, err := avmpkg.New(out)
+			if err != nil {
+				return fmt.Errorf("failed to initialize avm: %w", err)
+			}
 
-// AddCommand creates a new command to add a plugin
-func AddCommand(out output.Output) *cobra.Command {
-	return &cobra.Command{
-		Use:   "add",
-		Short: "Adds a plugin",
-		RunE: func(cmd *cobra.Command, args []string) error {
-			out.V(6).Info(fmt.Sprintf("args: %v", args))
-			return nil
-		},
-	}
-}
+			defaultSource := avm.GetDefaultSource()
+
+			err = defaultSource.InstallPluginVersion(
+				&types.InstallPluginVersionRequest{
+					Name:    "golang",
+					Version: "1.19.3",
```

Comment:
Do we want to have this hardcoded for now?

Reply:
Created a PR for this: https://github.com/d2iq-labs/avm/pull/18

Reply:
didn't want to spend time on flag config last night.  We could do it today